### ISSUE TITLE
Why? when i train faster-rcnn , it's global step view twice.

### ISSUE THREAD

**Issue by HalfLemon01** [tensorflow/models#4600]
###Situation
when i try object-detection  model
### Describe the problem
before today, the model and project is normal. however  my server exchange place. appear the problem.

### Source code / logs

INFO:tensorflow:Restoring parameters from /usr/local/tensorflow_models/models-master/research/object_detection/MODEL_FASTER/model.ckpt
INFO:tensorflow:Restoring parameters from /usr/local/tensorflow_models/models-master/research/object_detection/MODEL_FASTER/model.ckpt
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Starting Session.
INFO:tensorflow:Starting Session.
INFO:tensorflow:Saving checkpoint to path ./research/object_detection/TRAIN_EVAL/TRAIN_RESULT/model.ckpt
INFO:tensorflow:Saving checkpoint to path ./research/object_detection/TRAIN_EVAL/TRAIN_RESULT/model.ckpt
INFO:tensorflow:Starting Queues.
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:global step 1: loss = 2.1770 (10.038 sec/step)
INFO:tensorflow:global step 1: loss = 2.1770 (10.038 sec/step)
INFO:tensorflow:global step 2: loss = 1.7647 (0.673 sec/step)
INFO:tensorflow:global step 2: loss = 1.7647 (0.673 sec/step)
INFO:tensorflow:global step 3: loss = 1.8594 (1.599 sec/step)
INFO:tensorflow:global step 3: loss = 1.8594 (1.599 sec/step)
INFO:tensorflow:global step 4: loss = 1.7591 (0.168 sec/step)
INFO:tensorflow:global step 4: loss = 1.7591 (0.168 sec/step)
INFO:tensorflow:global step 5: loss = 1.4718 (0.150 sec/step)
INFO:tensorflow:global step 5: loss = 1.4718 (0.150 sec/step)
INFO:tensorflow:global step 6: loss = 1.4042 (0.123 sec/step)
INFO:tensorflow:global step 6: loss = 1.4042 (0.123 sec/step)
INFO:tensorflow:global step 7: loss = 1.4349 (0.145 sec/step)
INFO:tensorflow:global step 7: loss = 1.4349 (0.145 sec/step)
INFO:tensorflow:global step 8: loss = 1.1939 (0.120 sec/step)
INFO:tensorflow:global step 8: loss = 1.1939 (0.120 sec/step)
INFO:tensorflow:global step 9: loss = 1.2642 (0.127 sec/step)
INFO:tensorflow:global step 9: loss = 1.2642 (0.127 sec/step)

every step  run  twice.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by ashraful100** [comment]
Hi @HalfLemon01 !!!
Did you already solve this issue? I am also facing this problem. It's viewing twice for each global step.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
